specification/cognitiveservices/OpenAI.Inference/models/completions/chat_messages.tsp

```diff
@@ -73,17 +73,52 @@ union ChatMessageImageDetailLevel {
   high: "high",
 }
 
+/** A structured chat content item containing an image reference. */
 @added(ServiceApiVersions.v2024_02_15_Preview)
-@doc("A structured chat content item containing an image reference.")
 model ChatMessageImageContentItem extends ChatMessageContentItem {
-  @doc("The discriminated object type: always 'image_url' for this type.")
+  /** The discriminated object type: always 'image_url' for this type. */
   type: "image_url";
 
   @doc("An internet location, which must be accessible to the model,from which the image may be retrieved.")
   @encodedName("application/json", "image_url")
   imageUrl: ChatMessageImageUrl;
 }
 
+/** Values to describe the format of the input audio data. */
+@added(ServiceApiVersions.v2024_02_15_Preview)
+union InputAudioFormat {
+  string,
+
+  /** Specifies that the audio data is in the WAV format. */
+  wav: "wav",
+
+  /** Specifies that the audio data is in the MP3 format. */
+  mp3: "mp3",
+}
+
+/**
+ * A structured chat content item containing audio data.
+ */
+@added(ServiceApiVersions.v2025_01_01_Preview)
+model InputAudioContent {
+  /** Base64-encoded audio data. */
+  data: string;
```

Comment:
data: bytes;

Reply:
we talked about this, and we are keeping string here for consistency... the new spec has this properly set to bytes.